### PR TITLE
Small bpf cleanups

### PR DESCRIPTION
**Pull request by tklauser**
Please see individual commit messages.

### CODE REVIEW

**Comment by tklauser** [comment]
test-me-please

File: bpf/lib/ipv6.h [hunk 1]
```diff
@@ -116,7 +116,7 @@ static __always_inline int ipv6_addr_in_net(const union v6addr *addr,
 				&& (!mask->p4 || ((addr->p4 & mask->p4) == net->p4))))));
 }
 
-#define GET_PREFIX(PREFIX)						\
+#define GET_PREFIX(prefix)						\
 	bpf_htonl(prefix <= 0 ? 0 : prefix < 32 ? ((1<<prefix) - 1) << (32-prefix)	\
```
Comment (pchaigno) [inline bpf/lib/ipv6.h]: Nit: I think `prefix` should be capitalized instead, to indicate it's a macro argument. That's what we do elsewhere in the C code.

**Review by pchaigno — APPROVED** [review]
Nice!

File: bpf/lib/ipv6.h (hunk 1 above)
Comment (tklauser) [reply bpf/lib/ipv6.h]: Good point, done.

**Comment by tklauser** [comment]
test-me-please

**Review by borkmann — APPROVED** [review]

**Comment by tklauser** [comment]
retest-4.19

**Review by brb — APPROVED** [review]

**Review by joestringer — APPROVED** [review]

**Pull request merged**